saleor/settings.py

```diff
@@ -555,6 +555,10 @@ def get_host():
         "task": "saleor.csv.tasks.delete_old_export_files",
         "schedule": crontab(hour=1, minute=0),
     },
+    "update-products-search-vectors": {
+        "task": "saleor.product.tasks.update_products_search_vector_task",
+        "schedule": timedelta(seconds=20),
+    },
```

Comment:
What if it takes longer than 20s to execute the task? Won't it reindex multiple times the same products?

Reply:
@patrys any thoughts about this?

Reply:
It might happen that two tasks picked up the same products for an update, but if it happens for some small set of products and it reindexes them multiple times, would that be a big problem? Maybe we should actually measure how much time it usually takes and adjust the time delta?

Reply:
It will spawn a Celery task every 20 seconds. There is no guarantee that the task will even start executing within the 20-second window. I'm not aware of better alternatives as locking product rows is going to hurt business logic (it would block transactional code).

Reply:
so should we stay with this for now, check how it works and adjust later if it will be needed?